tools/gen_syscalls/gen_syscalls.sh

```diff
@@ -69,26 +75,75 @@ cat > "$outname" <<HEADER
 const struct syscall_descriptor ${arch,,}_syscall_list[] = {
 HEADER
 
-case "$arch" in
-ARM)
-	gdb --batch -ex 'set gnutarget elf32-littlearm' -ex "file $linux" -x extract.py
-	;;
-*)
-	gdb --batch -ex "file $linux" -x extract.py
-	;;
-esac
-
-if [ -f "missing/${arch,,}.c" ]; then
-	cat "missing/${arch,,}.c" >> output_syscalls.c
-fi
+	echo -n "" > output_syscalls.c
+
+
+	case "$arch" in
+	ARM)
+		SYSCALLTABLENAME="$SYSCALLTABLENAME" \
+		SYSCALLBIT=$SYSCALLBIT \
+			"$GDB" --batch \
+			-ex 'set gnutarget elf32-littlearm' \
+			-ex "file $linux" \
+			-x extract.py
+		;;
+	*)
+		SYSCALLTABLENAME="$SYSCALLTABLENAME" \
+		SYSCALLBIT=$SYSCALLBIT \
+			"$GDB" --batch \
+			-ex "file $linux" \
+			-x extract.py
+		;;
+	esac
 
-cat output_syscalls.c | sort -k1,1 --unique --stable -t',' >> "$outname"
-rm output_syscalls.c
+	if [ -f "missing/${arch,,}.c" ]; then
```

Comment:
would prefer that `missing/{arch}.c` takes precedence over automatically extracted as it was before